### ISSUE TITLE
Unclear that a second chart type cannot be the same as the first

### ISSUE THREAD

**Issue by cindyschen** [ramp4-pcar4/highcharts-accessible-configuration-kit#186]
## Description
Seems that you cannot make a combination chart with two of the same type of chart, and it's misleading in the templates tab

## Steps to reproduce

1. Upload any chart
2. Go to templates tab
3. Choose the same type of chart as your second chart
4. Nothing happens and the selection is erased if you switch tabs 

## Suggested fixes
Could just not have the option to select the same chart type as the second chart
